packages/manager/src/features/Managed/Contacts/ContactsDrawer.tsx

```diff
@@ -54,9 +53,17 @@ const ContactsDrawer = (props: ContactsDrawerProps) => {
 
   // If we're in Edit mode, take the initialValues from the contact we're editing.
   // Otherwise, all initial values should be empty strings.
+  const getContactInfo = (): ContactPayload => {
+    return {
+      email: contact?.email || '',
```

Comment:
It's preferable to use the [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here and below.
```suggestion
      email: contact?.email ?? '',
```